menas/ui/components/dataset/datasetDetail.view.xml

```diff
@@ -65,8 +65,9 @@
             <Bar>
                 <contentRight>
                     <Button id="Delete" text="Delete" icon="sap-icon://delete" press="onRemovePress" />
-                    <Button id="AddRule" text="Add Conformance Rule" icon="sap-icon://add-equipment" press="onAddConformanceRulePress"></Button>
-                    <Button id="Edit" text="Edit" icon="sap-icon://edit" press="onEditPress" />
+                    <Button id="AddRule" text="Add Conformance Rule" icon="sap-icon://add-equipment" press="onAddConformanceRulePress" />
+                    <Button id="Edit" text="Edit" icon="sap-icon://edit" press="onEditPress" enabled="{/editingEnabled}"
+                            tooltip="{= ${/editingEnabled} ? 'Editing enabled' : 'Editing old version is disabled'}" />
```

Comment:
There is something odd about `Editing enabled`, it's confusing to see when I expect to be able to edit. Makes sense to tell me why it's disabled, because it's an unexpected condition. 
For when it's enabled I'd expect a tooltip that tells me something like `Edit current dataset`.